serverless/serverless
# Add Google Python function template

## What did you implement:

Added a template for a Google Python function.

## How did you implement it:

Basically I just copied the Google NodeJS template, changed the runtime, added a "main.py" and ran it against my GCP account to verify it was working.

## How can we verify it:

After merge it should be possible to run:
```
serverless create --template google-python --path my-service
```
To create the service/function and then run the standard:
```
npm install
serverless deploy
serverless invoke --function first
```
to test it.

## Todos:


***Is this ready for review?:*** YES
***Is it a breaking change?:*** NO
